apps/dotcms-ui/src/app/view/components/_common/dot-action-button/dot-action-button.component.ts

```diff
@@ -44,13 +44,13 @@ export class DotActionButtonComponent implements OnInit, OnChanges {
     selected: boolean;
 
     @Output()
-    click: EventEmitter<any> = new EventEmitter();
+    press: EventEmitter<MouseEvent> = new EventEmitter();
 
     @HostBinding('class.action-button--no-label')
     isNotLabeled = true;
 
     @HostListener('click', ['$event'])
```

Comment:
change this to `press` ?

Reply:
`HostListener` is not the same as Output.